loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -150,17 +155,21 @@ DEFINE
  FALSE                     = 0
  DATENULL                  = '--/--/--'
 
- INELIGIBLECODESET         = 0
- INELIGIBLECODELOANTYPE    = 1
- INELIGIBLECODELOANWARNING = 2
- INELIGIBLECODEMAXSKIPS    = 3
- INELIGIBLECODETIMESKIP    = 4
- INELIGIBLECODEACCTWARNING = 5
- INELIGIBLECODESERVICECODE = 6
- INELIGIBLECODEMINPAYMENT  = 7
- INELIGIBLECODEMAXPAYMENT  = 8
- INELIGIBLECODETIMEOPEN    = 9
- INELIGIBLECODEPASTDUE     = 10
+ INELIGIBLECODESET         = 0  [loan ineligible flag]
+ INELIGIBLECODELOANTYPE    = 1  [Incorrect loan type]
+ INELIGIBLECODELOANWARNING = 2  [has loan warning]
+ INELIGIBLECODEMAXSKIPS    = 3  [has used max skips already]
+ INELIGIBLECODETIMESKIP    = 4  [not enough time between skips]
+ INELIGIBLECODEACCTWARNING = 5  [has account warning]
+ INELIGIBLECODESERVICECODE = 6  [does not have required svc code]
+ INELIGIBLECODEMINPAYMENT  = 7  [min payment requirement not met]
+ INELIGIBLECODEMAXPAYMENT  = 8  [max payment amount exceeded]
+ INELIGIBLECODETIMEOPEN    = 9  [loan not opened for sufficient time]
+ INELIGIBLECODEPASTDUE     = 10 [past due grace days exceeded]
+ INELIGIBLECODEFREQ        = 11 [has ineligibility payment freq]

```

Comment:
I think "has ineligibility", should be "has ineligible"

Reply:
Changed